src/devices/Card/CreditCard/ApduCommands.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// The list of predefined commands to communicate with the card
+    /// </summary>
+    public class ApduCommands
```

Comment:
these should be either internal or create new instance of array each time or be IEnumerable - otherwise modification of these will have effect on the code